Dockerfile

```diff
@@ -67,13 +67,16 @@ RUN ckan-pip install -r https://raw.githubusercontent.com/ckan/ckanext-scheming/
 RUN ckan-pip install git+https://github.com/data-govt-nz/ckanext-security.git Beaker==1.6.4
 
 # S3 filestore extension
-RUN ckan-pip install git+https://github.com/okfn/ckanext-s3filestore@v0.1.1 boto3>=1.4.4 ckantoolkit
+RUN ckan-pip install git+https://github.com/okfn/ckanext-s3filestore boto3>=1.4.4 ckantoolkit
```

Comment:
I would prefer we stick with a tagged release, since we're pulling from the `s3filestore` repo directly.